webrender/src/device/gfx/buffer.rs

```diff
@@ -5,12 +5,14 @@
 use hal;
 use hal::Device as BackendDevice;
 use rendy_memory::{Block, Heaps, MemoryBlock, MemoryUsageValue, Write};
+use smallvec::SmallVec;
 
 use std::cell::Cell;
 use std::mem;
 
-pub const MAX_INSTANCE_COUNT: usize = 8192;
-pub const TEXTURE_CACHE_SIZE: usize = 128 << 20; // 128MB
+pub const INSTANCE_BUFFER_SIZE: usize = 1 << 20; // 1MB
```

Comment:
we should make this configurable in RendererOptions

Reply:
I've made this and the texture cache size configurable in 91f6fc33a985bc186d77ae97bbd10c9d78b81f60